### PR TITLE
STYLE: New OpenVR Logo

### PR DESCRIPTION
**Pull request by jbvimort**

### CODE REVIEW

**Comment by jcfr** [comment]
Thanks.

* [ ] Instead of `VR.png`. Could you add the icon at the root of the project and renamed it `SlicerOpenVR.png`
* [ ] Could you look into having a border, and light background ? These may help https://github.com/SlicerIGT/SlicerIGT/blob/master/SlicerIGTLogo.png and https://github.com/SlicerIGT/SlicerIGT/blob/master/SlicerIGTLogoDesign.xcf

**Comment by jbvimort** [comment]
Should look better now! :smiley:

File: CMakeLists.txt [hunk 1]
```diff
@@ -8,7 +8,7 @@ set(EXTENSION_HOMEPAGE "http://slicer.org/slicerWiki/index.php/Documentation/Nig
 set(EXTENSION_CATEGORY "Examples")
 set(EXTENSION_CONTRIBUTORS "Jean-Baptiste Vimort (Kitware), Jean-Christophe Fillion-Robin (Kitware)")
 set(EXTENSION_DESCRIPTION "This extension allow to use a an HTC vive to display data in Virtual Reallity")
-set(EXTENSION_ICONURL "http://www.example.com/Slicer/Extensions/VR.png")
+set(EXTENSION_ICONURL "https://rawgit.com/KitwareMedical/SlicerOpenVR/master/VR.png")
```
Comment (jcfr) [inline CMakeLists.txt]: I don't think rawgit.com is required, the Content-Type for the url like seems correct https://raw.githubusercontent.com/KitwareMedical/SlicerOpenVR/master/VR.png

Comment (jbvimort) [reply CMakeLists.txt]: Changed! :smiley_cat:

**Comment by jcfr** [comment]
🎆

**Pull request merged**